### PR TITLE
Amplitude product fix

### PR DESCRIPTION
**Pull request by sajalmohanta**

### CODE REVIEW

File: v0/destinations/am/data/AmplitudeRevenueConfig.json [hunk 1]
```diff
@@ -9,11 +9,11 @@
   "context.app.build": "app_version",
   "context.device.type": "platform",
   "context.locale": "language",
-  "properties.productId": "productId",
+  "properties.product_Id": "productId",
```
Comment (arnab-p) [inline v0/destinations/am/data/AmplitudeRevenueConfig.json:12]: I think it should be all lower `product_id`. 

Ref: https://docs.rudderstack.com/rudderstack-api-spec/rudderstack-ecommerce-events-specification/ordering#product-viewed

File: v0/destinations/am/data/AmplitudeRevenueConfig.json [hunk 2]
```diff
@@ -9,11 +9,11 @@
...
   "properties.price": "price",
   "properties.quantity": "quantity",
   "properties.revenue": "revenue",
-  "properties.revenueType": "revenueType",
+  "properties.revenue_Type": "revenueType",
```
Comment (arnab-p) [inline v0/destinations/am/data/AmplitudeRevenueConfig.json:16]: Similar as above

File: v0/destinations/am/data/AmplitudeRevenueConfig.json (hunk 1 above)
Comment (sajalmohanta) [reply v0/destinations/am/data/AmplitudeRevenueConfig.json]: Updated

File: v0/destinations/am/data/AmplitudeRevenueConfig.json (hunk 2 above)
Comment (sajalmohanta) [reply v0/destinations/am/data/AmplitudeRevenueConfig.json]: Updated

**Comment by arnab-p** [comment]
Resolves: 
https://github.com/rudderlabs/rudder-transformer/issues/197
https://github.com/rudderlabs/rudder-transformer/issues/198

**Pull request merged**